$OpenBSD: patch-tests_threads_cc,v 1.1.1.1 2013/03/22 13:57:48 ajacoutot Exp $

tests/threads.cc:31: error: invalid conversion from 'pthread*' to 'int'

--- tests/threads.cc.orig	Mon Jan 14 09:24:17 2013
+++ tests/threads.cc	Mon Jan 14 09:24:23 2013
@@ -28,7 +28,7 @@ thread_func (void *arg)
 {
     assert (arg != 0);
     thread_info *ti = static_cast<thread_info*> (arg);
-    __attribute__((unused)) int tid = ti->tid;
+    __attribute__((unused)) pthread_t tid = ti->tid;
     return NULL;
 }
 
